lasagne/layers/autoenc.py

```diff
@@ -0,0 +1,262 @@
+import theano
+
+from .. import nonlinearities
+from .. import init
+
+from .base import MergeLayer
+from .helper import get_all_layers
+from .input import InputLayer
+from .special import NonlinearityLayer, BiasLayer
+from .noise import DropoutLayer, GaussianNoiseLayer
+
+
+__all__ = [
+    "InverseLayer",
+    "build_autoencoder",
+    "unfold_bias_and_nonlinearity_layers",
+]
+
+
+class InverseLayer(MergeLayer):
+    """
```

Comment:
It would be good to have a short first sentence in the beginning, since that's used for overviews such as the tables here: http://lasagne.readthedocs.org/en/latest/modules/layers.html